### ISSUE TITLE
something wrong when run main.py

### ISSUE THREAD

**Issue by RunChengNi** [zhutmost/lsq-net#7]
File "/home/chengrun/lsq-net-master/main.py", line 106, in <module>
    main()
  File "/home/chengrun/lsq-net-master/main.py", line 19, in main
    log_dir = util.init_logger(args.name, output_dir, 'logging.conf')
  File "/home/chengrun/lsq-net-master/util/config.py", line 46, in init_logger
    logging.config.fileConfig(cfg_file, defaults={'logfilename': log_file})
  File "/home/chengrun/.conda/envs/lsq/lib/python3.8/logging/config.py", line 71, in fileConfig
    formatters = _create_formatters(cp)
  File "/home/chengrun/.conda/envs/lsq/lib/python3.8/logging/config.py", line 104, in _create_formatters
    flist = cp["formatters"]["keys"]
  File "/home/chengrun/.conda/envs/lsq/lib/python3.8/configparser.py", line 960, in __getitem__
    raise KeyError(key)
KeyError: 'formatters'
can tell me where is the problem?

**Comment by zhutmost** [comment]
Please double check your Conda environment. Do you install all the dependencies with the correct versions?

```bash
# Create a environment with Python 3.8
conda create -n lsq python=3.8
# PyTorch GPU version >= 1.5
conda install pytorch torchvision cudatoolkit=10.1 -c pytorch
# Tensorboard visualization tool
conda install tensorboard
# Miscellaneous
conda install scikit-learn pyyaml munch
```

**Comment by RunChengNi** [comment]
yeah,i checked it,maybe&nbsp; the path of log is wrong




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "zhutmost/lsq-net"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年7月17日(星期五) 下午4:14
收件人:&nbsp;"zhutmost/lsq-net"<lsq-net@noreply.github.com&gt;;
抄送:&nbsp;"抓小姐姐玩"<549772655@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [zhutmost/lsq-net] something wrong when run main.py (#7)





 
Please double check your Conda environment. Do you install all the dependencies with the correct versions?
 # Create a environment with Python 3.8 conda create -n lsq python=3.8 # PyTorch GPU version &gt;= 1.5 conda install pytorch torchvision cudatoolkit=10.1 -c pytorch # Tensorboard visualization tool conda install tensorboard # Miscellaneous conda install scikit-learn pyyaml munch

**Comment by zhutmost** [comment]
Please execute `python main.py ...` in the root folder of the repository.

**Comment by RunChengNi** [comment]
python /home/chengrun/lsq-net-master/main.py /home/chengrun/lsq-net-master/config.yaml

&nbsp;is this right?




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "zhutmost/lsq-net"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年7月17日(星期五) 下午4:18
收件人:&nbsp;"zhutmost/lsq-net"<lsq-net@noreply.github.com&gt;;
抄送:&nbsp;"抓小姐姐玩"<549772655@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [zhutmost/lsq-net] something wrong when run main.py (#7)





 
Please execute python main.py ... in the root folder of the repository.

**Comment by zhutmost** [comment]
```
cd /home/chengrun/lsq-net-master
python main.py your_config.yaml
```
Do NOT change the content of the default config.yaml.

**Comment by RunChengNi** [comment]
python /home/chengrun/lsq-net-master/main.py /home/chengrun/lsq-net-master/myconfig.yaml



File "/home/chengrun/lsq-net-master/main.py", line 106, in <module&gt;
&nbsp; &nbsp; main()
&nbsp; File "/home/chengrun/lsq-net-master/main.py", line 19, in main
&nbsp; &nbsp; log_dir = util.init_logger(args.name, output_dir, 'logging.conf')
&nbsp; File "/home/chengrun/lsq-net-master/util/config.py", line 46, in init_logger
&nbsp; &nbsp; logging.config.fileConfig(cfg_file, defaults={'logfilename': log_file})
&nbsp; File "/home/chengrun/.conda/envs/lsq/lib/python3.8/logging/config.py", line 71, in fileConfig
&nbsp; &nbsp; formatters = _create_formatters(cp)
&nbsp; File "/home/chengrun/.conda/envs/lsq/lib/python3.8/logging/config.py", line 104, in _create_formatters
&nbsp; &nbsp; flist = cp["formatters"]["keys"]
&nbsp; File "/home/chengrun/.conda/envs/lsq/lib/python3.8/configparser.py", line 960, in __getitem__
&nbsp; &nbsp; raise KeyError(key)
KeyError: 'formatters'



still happened.
------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "zhutmost/lsq-net"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年7月17日(星期五) 下午4:30
收件人:&nbsp;"zhutmost/lsq-net"<lsq-net@noreply.github.com&gt;;
抄送:&nbsp;"抓小姐姐玩"<549772655@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [zhutmost/lsq-net] something wrong when run main.py (#7)





 cd /home/chengrun/lsq-net-master python main.py your_config.yaml  
Do NOT change the content of the default config.yaml.

**Comment by RunChengNi** [comment]
ok,i know,thanks!




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "zhutmost/lsq-net"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年7月18日(星期六) 下午2:44
收件人:&nbsp;"zhutmost/lsq-net"<lsq-net@noreply.github.com&gt;;
抄送:&nbsp;"抓小姐姐玩"<549772655@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [zhutmost/lsq-net] something wrong when run main.py (#7)





 
Closed #7.
